ui/ui-components/components/lib/entity/EntityList.tsx

```diff
@@ -27,16 +28,27 @@ const renderNameList = function <T extends Models.EntityTypes>(
       ? "overview"
       : "edit";
   const itemPath = itemType === "schedule" ? "source" : itemType;
+  const style: React.CSSProperties = {
+    verticalAlign: "middle",
+    marginTop: "-0.25em",
+    marginRight: 0,
+    marginBottom: 0,
+  };
```

Comment:
Nitpick, could be a class.

Reply:
I tried fixing this but one issue with the `<StateBadge>` is that it has hard-coded styles for the margins, so assigning a class will not override margins. Right now it's simpler to assign a style.